.github/workflows/docker-build.yml

```
@@ -0,0 +1,50 @@
+    - name: Checkout code
+      uses: actions/checkout@v4
+    - name: Pass SHA and Is_Release to Dockerfile Args
+      run: |
+        if [[ "${{ startsWith(github.ref, 'refs/tags/') }}" == "true" ]]; then
+          echo "IS_RELEASE=true" >> $GITHUB_ENV
+        else
+          echo "IS_RELEASE=false" >> $GITHUB_ENV
+        fi
+    - name: Get LedFx Version and Upload to Artifacts for later use
+      id: ledfx-version
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:20: Quote this to prevent word splitting

(shellcheck)